.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/gnosis/safe-contracts"]
+	path = external/gnosis/safe-contracts
+	url = git@github.com:OpenSTFoundation/safe-contracts.git
```

Comment:
probably needs to be checked out with `http`;

Reply:
Thx. Done.